docs/en/relying-party-endpoint.rst

```diff
@@ -0,0 +1,242 @@
+    :name: code_RelyingParty_Endpoint_IntegrityValidation_Error
+    
+    HTTP/1.1 403 Forbidden
+    Content-Type: application/json
+
+    {
+        "error": "forbidden",
+        "error_description": "The provided challenge is invalid, expired, or already used."
+    }
+
+The following table lists HTTP Status Codes and related error codes that MUST be supported for the error response, unless otherwise specified:
+
+.. list-table::
+    :widths: 30 20 50
+    :header-rows: 1
+
+    * - **HTTP Status Code**
+      - **Error Code**
+      - **Description**
+    * - ``400 Bad Request``
+      - ``bad_request``
+      - The request is malformed, missing required parameters (e.g., header parameters or integrity assertion), or includes invalid and unknown parameters.
+    * - ``403 Forbidden`` 
+      - ``integrity_check_error``
+      - The device does not meet the Relying Party's minimum security requirements.
+    * - ``403 Forbidden``
+      - ``invalid_request``
+      - The signature of the Integrity Validation Request is invalid or does not match the associated public key (JWK).
+    * - ``403 Forbidden`` 
+      - ``invalid_request``
+      - The integrity assertion validation failed; the integrity assertion is tampered with or improperly signed.
+    * - ``403 Forbidden`` 
+      - ``invalid_request``
+      - The provided challenge is invalid, expired, or already used.
+    * - ``403 Forbidden``
+      - ``invalid_request``
+      - The Proof of Possession (``hardware_signature``) is invalid.
+    * - ``403 Forbidden`` 
+      - ``invalid_request``
+      - The ``iss`` parameter does not match the Relying Party expected URL identifier.
+    * - ``404 Not Found`` 
+      - ``not_found``
+      - The Relying Party Instance was not found.
+    * - ``422 Unprocessable Content`` [OPTIONAL]
+      - ``validation_error``
+      - The request does not adhere to the required format.
+    * - ``500 Internal Server Error``
+      - ``server_error``
+      - The request cannot be fulfilled because the Endpoint encountered an internal problem.
+    * - ``503 Service Unavailable``
+      - ``temporarily_unavailable``
+      - The request cannot be fulfilled because the Endpoint is temporarily unavailable (e.g., due to maintenance or overload).
+
+
+Relying Party Access Certificate Endpoint
+----------------------------------------------
+
+The Relying Party Access Certificate Endpoint enables Relying Party Instances to obtain an Access Certificate.
+
+
+Relying Party Access Certificate Request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The request uses the HTTP POST method with ``Content-Type`` set to ``application/json``.
+
+The Access Certificate Request includes the following body parameters:
+
+.. list-table::
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Parameter**
+      - **Description**
+      - **Reference**
+    * - **csr**
+      - The CSR generated by the Relying Party Instance, encoded in the ``base64url`` format.
```

Comment:
CSR must be included within the acronyms section, and rereferencing [rfc 5280](https://datatracker.ietf.org/doc/html/rfc2511)